docs/Outbound.md

```diff
@@ -348,16 +348,17 @@ should queueing to disk fail e.g:
 
     outbound.send_email(from, to, contents);
 
+You can pass various options to `outbound.send_email` like so:
 
-In case you are passing your content dot-stuffed (a dot at the start of a line
-is doubled, like it is in SMTP conversation, 
-see https://tools.ietf.org/html/rfc2821#section-4.5.2), you should pass the
-```dot_stuffed: true``` option, like so:
-    
-    outbound.send_email(from, to, contents, outnext, { dot_stuffed: true });
+    outbound.send_email(from, to, contents, outnext, options);
+
+Where `options` is a Object that may contain the following keys:
 
+| Key/Value              | Description                                                                               |
+|------------------------|--------------------------------------------------------------------------------------------
+| `dot_stuffed: true`    | Use this if you are passing your content dot-stuffed (a dot at the start of a line is doubled, like it is in SMTP conversation, see https://tools.ietf.org/html/rfc2821#section-4.5.2).|
```

Comment:
I realize you didn't add it but updating this to be a link: `[RFC 2821](https://tools.ietf.org/html/rfc2821#section-4.5.2)` would improve it.